02-peer-protocol.md

```diff
@@ -192,6 +192,10 @@ know this node will accept `funding_satoshis` greater than or equal to 2^24.
 Since it's broadcast in the `node_announcement` message other nodes can use it to identify peers 
 willing to accept large channel even before exchanging the `init` message with them. 
 
+Implementers are advised to provide the means to scale the number of confirmations, tweaking 
+`accept_channel.minimum_depth`, with the size of the funding amount. A rule of thumb is to 
+wait enough blocks until the cumulative block reward exceeds the size of the channel.
```

Comment:
Maybe we should also recommend to scale `to_self_delay` in consequence ? You may have watchtowers to alleviate the liveliness requirement but mempool congestion is still a concern.

Reply:
Why does the issue apply to `to_self_delay`? I don't think it does, I think only funding is important as you don't want to start using the channel, relay payments and then see the channel's funding tx be double spent. But what could the attack be when closing?

Reply:
Eclipsing your node and slowing block your block reception. That way I pin you with a _delayed_ view of blockchain view, can broadcast a revoked commitment transaction and you won't react until it's too late. An adversary would also beneficiate from a congestioned mempool for your justice txn getting confirmed. This attack may be hard to setup but for a high-value channel they shouldn't be excluded, increasing `to_self_delay` means increasing period during which eclipsing must be sustained.